File: payments/JS/mangopay.js
import mangopay from 'mangopay2-nodejs-sdk';

async function main(){
    var api = new mangopay({
        clientId: 'your_client_id',
        clientApiKey: 'your_client_api_key',
        // Set the right production API url. If testing, omit the property since it defaults to sandbox URL
        baseUrl: 'https://api.mangopay.com'
    });

    api.Users.create()
}

File: proj dotnet/Dotnet/mangopay.cs
using MangoPay.SDK;
using MangoPay.SDK.Entities;
using MangoPay.SDK.Core.Enumerations;

class Program
{
    static void Main()
    {
        var api = new MangoPayApi();
        api.Config.ClientId = "YourClientId";
        api.Config.ClientPassword = "YourClientPassword";
        api.Config.BaseUrl = "https://api.sandbox.mangopay.com"; // or production URL

        // Use the API here for various operations
        // For example, creating a user:
        var user = new UserNatural
        {
            FirstName = "John",
            LastName = "Doe",
            Email = "john.doe@example.com",
            Birthday = 636467136000000000, // Representing a birth date, adjust as needed
            Nationality = CountryIso.FR,
            CountryOfResidence = CountryIso.FR
        };

        var createdUser = api.Users.Create(user);
    }
}
